contracts/modules/token/TokenController.sol

```diff
@@ -387,25 +387,30 @@ contract TokenController is ITokenController, LockHandler, MasterAwareV2 {
     return StakingPoolLibrary.getAddress(stakingPoolFactory, poolId);
   }
 
-  function mintStakingPoolNXMRewards(uint amount, uint poolId) external {
+  function mintStakingPoolNXMRewards(uint amount, uint poolId) external whenNotPaused {
     require(msg.sender == _stakingPool(poolId), "TokenController: msg.sender not staking pool");
     token().mint(address(this), amount);
     stakingPoolNXMBalances[poolId].rewards += amount.toUint128();
   }
 
-  function burnStakingPoolNXMRewards(uint amount, uint poolId) external {
+  function burnStakingPoolNXMRewards(uint amount, uint poolId) external whenNotPaused {
     require(msg.sender == _stakingPool(poolId), "TokenController: msg.sender not staking pool");
     stakingPoolNXMBalances[poolId].rewards -= amount.toUint128();
     token().burn(amount);
   }
 
-  function depositStakedNXM(address from, uint amount, uint poolId) external {
+  function depositStakedNXM(address from, uint amount, uint poolId) external whenNotPaused {
     require(msg.sender == _stakingPool(poolId), "TokenController: msg.sender not staking pool");
     stakingPoolNXMBalances[poolId].deposits += amount.toUint128();
     token().operatorTransfer(from, amount);
   }
 
-  function withdrawNXMStakeAndRewards(address to, uint stakeToWithdraw, uint rewardsToWithdraw, uint poolId) external {
+  function withdrawNXMStakeAndRewards(
```

Comment:
These are redundant since the calling functions already check for emergency paused

Reply:
good point, fixed